yarn.lock

```diff
@@ -4538,8 +4538,8 @@ __metadata:
   linkType: hard
 
 "@metamask/providers@npm:^11.0.0":
-  version: 11.1.0
-  resolution: "@metamask/providers@npm:11.1.0"
+  version: 11.0.0
```

Comment:
Why the revert 🤔

Reply:
Not intentional, not sure why this happened.